controllers/shops.js

```diff
@@ -0,0 +1,29 @@
+const { isEmpty } = require("lodash");
+const { shops } = require("../models");
+const moment = require("moment");
+
+module.exports = {
+  create: async (req, res) => {
+    try {
+      const { name, phoneNumber, address } = req.body;
+      const User = req.user;
+      if (isEmpty(name) || isEmpty(phoneNumber) || isEmpty(address)) {
+        throw { status: 400, message: "Required fields cannot be empty" };
+      }
+      const isShop = await shops.findOne({ where: { fk_user_id: User.id } });
+      if (isShop) {
+        throw { status: 401, message: "Shop Already Exists" };
+      }
+      const Shop = await shops.create({
```

Comment:
```suggestion
      Shop = await shops.create({
```
As above you already declared `Shop` as per my suggestion.